src/App.tsx

```diff
@@ -10,33 +10,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is&nbsp;
```

Comment:
Your tests are failing. I believe it is somehow connected to this &nbsp this. Try to delete it and just write everything in one line here
```suggestion
        {`Selected tab is ${selectedTab.title}`}
```